cmd/vclusterctl/cmd/delete.go

```diff
@@ -153,6 +154,28 @@ func (cmd *DeleteCmd) Run(cobraCmd *cobra.Command, args []string) error {
 		} else {
 			cmd.log.Donef("Successfully deleted virtual cluster namespace %s", cmd.Namespace)
 		}
+
+		// delete multi namespace mode namespaces
```

Comment:
Should we mention this new functionality in the description of the `--delete-namespace` flag?

Reply:
Yeah I'll add a mention!